### PR TITLE
2016Q4.feature.gateway_v6

### PR DESCRIPTION
**Pull request by fgiaimo**
Changes with the new dbc file: 

New CAN/ODVD Message:
EnergyLevels

Renamed CAN/ODVD Messages:
SteeringRequest => SteerRequest
ManualControl => ManualDriving
Propulsion => VehicleSpeed
VehicleState => VehicleDynamics

Renamed File: 
fh16gw.dbc => fh16gw_v6.dbc

Some of these changes can break the code on other branches that don't use the updated names for the CAN/ODVD messages (master builds, however).

### CODE REVIEW

File: resources/odvd/ODVDFH16Truck.odvd [hunk 1]
```diff
@@ -16,11 +16,12 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-message opendlv.proxy.reverefh16.ManualControl [id = 191] {
+message opendlv.proxy.reverefh16.ManualDriver [id = 191] {
```
Comment (chrberger) [inline resources/odvd/ODVDFH16Truck.odvd:19]: As this PR contains renaming of messages, please make sure that you propagate this renaming to the higher-level repositories

**Review by chrberger — CHANGES_REQUESTED** [review]

File: resources/odvd/ODVDFH16Truck.odvd (hunk 1 above)
Comment (chrberger) [reply resources/odvd/ODVDFH16Truck.odvd:19]: @fgiaimo What is the status of this PR?

Comment (fgiaimo) [reply resources/odvd/ODVDFH16Truck.odvd:19]: This PR is connected to https://github.com/chalmers-revere/opendlv/pull/119 . 
Approving both should fix the inter-dependencies among the two repositories.

**Review by olbender — APPROVED** [review]
This is fine. But we should soon move the Rhino related code to its own repository.

File: resources/odvd/ODVDFH16Truck.odvd (hunk 1 above)
Comment (fgiaimo) [reply resources/odvd/ODVDFH16Truck.odvd:19]: as support_for_dbc_v6 was merged in opendlv, we will need this PR merged as well to solve the naming changes

**Review by chrberger — APPROVED** [review]

**Pull request merged**